glide.yaml

```diff
@@ -26,9 +26,7 @@ import:
   subpackages:
   - pkg/util/validation
 - package: github.com/envoyproxy/data-plane-api
-  version: 28d28198e583aec75b6242ee02682654197b745e
-  subpackages:
-  - api/auth
+  version: master
```

Comment:
could we pin it to a specific `sha`. The upstream guys are re-organizing the layout for `authz` and it would break us once those PR's get merged.